elastic/eui
# [EuiDatePicker] Fix missing prop type for popperProps

## Summary

This PR adds a prop type for `popperProps` on `EuiDatePicker` as it's being used in the component to pass along to the `EuiPopover` component, but it's not being defined as prop.

## QA

- checkout the PR `gh pr checkout 7694`
- pass `popperProps` on a usage of `EuiDatePicker`, e.g. in `src-docs/src/views/date_picker/range.tsx`
- observe the prop not being flagged as TS error (IDE or via `yarn lint`)

### General checklist

- Release checklist
